### ISSUE TITLE
AttributeError: can't set attribute when using pre-trained extractive summariser

### ISSUE THREAD

**Issue by bmanczak** [HHousen/TransformerSum#46]
Hi,

Great work on the repo. I followed the [Getting Started](https://transformersum.readthedocs.io/en/latest/general/getting-started.html) page and tried to run `mobilebert-uncased-ext-sum` model. Here is a simple code snippet I used:

```
import os
import sys

sys.path.insert(0, os.path.abspath("./src"))
from extractive import ExtractiveSummarizer

model = ExtractiveSummarizer.load_from_checkpoint("mobilebert-uncased-ext-sum.ckpt")

text_to_summarize = "my long text."
 
model.predict(text_to_summarize) 
```

However, I get the following traceback

```
Traceback (most recent call last):
  File "/Users/blazejmanczak/Desktop/Projects/Artemos/ext_summarization/transformersum/testing_extractive.py", line 8, in <module>
    model = ExtractiveSummarizer.load_from_checkpoint("mobilebert-uncased-ext-sum.ckpt")
  File "/opt/miniconda3/envs/transformersum/lib/python3.9/site-packages/pytorch_lightning/core/saving.py", line 157, in load_from_checkpoint
    model = cls._load_model_state(checkpoint, strict=strict, **kwargs)
  File "/opt/miniconda3/envs/transformersum/lib/python3.9/site-packages/pytorch_lightning/core/saving.py", line 199, in _load_model_state
    model = cls(**_cls_kwargs)
  File "/Users/blazejmanczak/Desktop/Projects/Artemos/ext_summarization/transformersum/src/extractive.py", line 109, in __init__
    self.hparams = hparams
  File "/opt/miniconda3/envs/transformersum/lib/python3.9/site-packages/torch/nn/modules/module.py", line 995, in __setattr__
    object.__setattr__(self, name, value)
AttributeError: can't set attribute
```

Any tips on how to solve that?

**Comment by HHousen** [comment]
I've narrowed the problem down to this commit in pytorch-lightning https://github.com/PyTorchLightning/pytorch-lightning/pull/6207. Try running after installing the previous version of pytorch-lightning with `pip install -U pytorch_lightning==1.2.10`. I'm working on fixing the code to work with the latest version of pytorch-lightning.

**Comment by HHousen** [comment]
For reference, I found an open issue with this exact problem: https://github.com/PyTorchLightning/pytorch-lightning/issues/7443#issuecomment-836983666

**Comment by HHousen** [comment]
By the way, you're going to need to set `strict=False` when calling `load_from_checkpoint` with that model.

**Comment by HHousen** [comment]
Solved in e36e0331f34427973403b4c340cc364fcf9fe26b

**Comment by bmanczak** [comment]
Thank you for a swift patch! 

To make it work I had to do one small thing: 
In `src/extractive.py` change `nlp.add_pipe(sentencizer)` to  `nlp.add_pipe("sentencizer")`.
